packages/shared/src/utils/constants/mt5-login-list-status.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * this comes from mt5_login_list endpoint
+ */
 export const MT5_ACCOUNT_STATUS = Object.freeze({
```

Comment:
Suggestion: Use `as const` instead of `Object.freeze` to ensure type safety and immutability at compile-time for both `MT5_ACCOUNT_STATUS` and `TRADING_PLATFORM_STATUS`.